test/metabase/query_processor_test/explicit_joins_test.clj

```diff
@@ -575,6 +575,6 @@
                                               [:field "count" {:base-type :type/BigInteger, :join-alias "Q2"}]
                                               [:field "count" {:base-type :type/BigInteger}]]}
                          :limit        2})]
-            (is (= [[4 41 0.46 41]]
+            (is (= [[4 89 0.46 89]]
```

Comment:
The actual correct result is `[4 89 0.46 41]`, but it's too complicated to fix in this PR. (So this is just changing from one wrong result to another wrong result.)

Opened #18086 to track the actual issue